src/main/java/com/impactupgrade/nucleus/service/segment/MailchimpCommunicationService.java

```diff
@@ -421,6 +424,12 @@ protected MemberInfo toMcMemberInfo(CrmContact crmContact, Map<String, Object> c
     mcContact.merge_fields.mapping.put(FIRST_NAME, crmContact.firstName);
     mcContact.merge_fields.mapping.put(LAST_NAME, crmContact.lastName);
     mcContact.merge_fields.mapping.put(PHONE_NUMBER, crmContact.mobilePhone);
+
+    mcContact.consents_to_one_to_one_messaging = true; //?
+    mcContact.sms_phone_number = crmContact.mobilePhone;
```

Comment:
@VSydor Update this to use `crmContact.phoneNumberForSMS()` instead -- it attempts Home Phone if Mobile doesn't exist.

Reply:
But is it possible to send sms to a home phone?

Reply:
Clients aren't all that consistent when they're entering phone numbers, so a cell number will often land in Home Phone instead of Mobile Phone. Ditto for web form integrations. So, we just try both...